openfl-tutorials/interactive_api/PyTorch_BrainMRI_UNet/envoy_1/shard_descriptor.py

```diff
@@ -0,0 +1,193 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
@saransh09 could we add type annotations? Like here - https://github.com/intel/openfl/blob/develop/openfl-tutorials/interactive_api/PyTorch_Lightning_MNIST_GAN/envoy/mnist_shard_descriptor.py